tests/sles4sap/qesapdeployment/test_cluster.pm

```diff
@@ -9,7 +9,7 @@ use warnings;
 use Mojo::Base 'publiccloud::basetest';
 use testapi;
 use qesapdeployment;
-use hacluster '$crm_mon_cmd';
+use hacluster;
```

Comment:
if you have issue with double definition of `post_fail_hook` (that I do not understand why it is happening and why only for this file and not for all the others that has `use hacluster;`, you can maybe try by changing this line to 

```
use hacluster qw($crm_mon_cmd cluster_check_status_by_regex);
```

Reply:
ok